near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -305,13 +305,40 @@ impl NonFungibleToken {
 
     /// Mint a new token without checking:
     /// * Whether the caller id is equal to the `owner_id`
+    /// * Assumes there will be a refund to the owner after covering the storage costs
```

Comment:
Ah do you mind updating the comment to the predecessor, if not nbd I can do after this comes in

Reply:
```suggestion
    /// * Assumes there will be a refund to the predecessor after covering the storage costs
```